lib/elliptic/ec/index.js

```diff
@@ -97,7 +98,7 @@ EC.prototype.sign = function sign(msg, key, enc, options) {
     options = {};
 
   key = this.keyFromPrivate(key, enc);
```

Comment:
what if we just did this
`var msgBitSize = typeof msg === 'string' ?  msg.replace(/\s+/g,'').length * 4  :  64;`

leave the public function prototypes as they are?